### ISSUE TITLE
[NPC] Blackheart the Inciter - [Spell] Incite Chaos' malfunctions with hunters

### ISSUE THREAD

**Issue by mhdemn** [Atlantiss/NetherwingBugtracker#2625]
**Description**: During this boss fight, [Blackheart the Inciter](https://db.excalibur.ws/?npc=18667) (here's [heroic](https://db.excalibur.ws/?npc=20637) ver) will cast [Incite Chaos](https://db.excalibur.ws/?spell=33676) now and then, putting a [debuff with the same name](https://db.excalibur.ws/?spell=33684) on all group members that will make them fight each other during 15 seconds. Also casts War Stomp.

**Current behaviour**: Every mechanic works fine and is casted with a blizzlike (It seems) frequence. However, the [Incite Chaos](https://db.excalibur.ws/?spell=33684)) debuff will make hunters affected by it use melee attacks only, instead of using all kind of skills in a smart way. 

Also, the debuff is showing in the buff bar. You can't click it off anyway, it's merely a visual thing.

**Expected behaviour**: You can see in this potato quality [video from retail TBC](https://www.youtube.com/watch?v=kTQefSWeFHY) that the hunter does use his ranged attacks and all kind of abilities during mind control; it's smart enough to use ranged attacks on far away targets, but not that much to kite and keep shooting when target gets in melee range, so then it starts using melee attacks.

You can also see Incite Chaos visually appears as a debuff.

**Server Revision**: 2304

**Comment by Mikik3jr** [comment]
Our hunter was draining my mana with his viper sting during that fight on HC.

**Comment by mhdemn** [comment]
I'm not sure where I can check that list Wolf. I'll check the retail video and see which ones he uses if you need me to.

Also, I'd say Hakkar's mind control makes hunter casts same sort of abilities as this one, maybe that can be a possible source?

What I observed during the fight, and I'll make sure to record it next time, is that unlike the hunter in the retail video, my toon just targeted the closest party member and run towards him to melee attack him. Didn't pay much attention to the spells casted, had to tab in and out during that fight.

Guessing his ability list for mind controls is correct, except for Auto Shot not getting prio when target is 3-35 yards away?
Something like that I'd say, since you're saying he's casting whatever he can cast during the time he's ranged, before running into melee.

**Comment by mhdemn** [comment]
Updating with another malfunction related to this spell, I thought I added it 6 days ago but I realized today I didn't.

Currently 2321:

After this bossfight, the Incite Chaoses (presumably) previously casted leave warriors bugged with some weird response to charging until they restart their client.

Whether it's Charge, the in-combat charge or Intervene, the warrior will finish it's pathing and then start walking up to some group member (Chosen randomly it seems). The warrior won't be able to stop this, and is completely locked out of his character's control until it completes this short walk.

*By walking I mean running at normal character speed, not actually roleplay walking
